solr/core/src/test/org/apache/solr/cloud/TestTolerantUpdateProcessorCloud.java

```diff
@@ -95,10 +95,9 @@ public class TestTolerantUpdateProcessorCloud extends SolrCloudTestCase {
   public static void createMiniSolrCloudCluster() throws Exception {
 
     final String configName = "solrCloudCollectionConfig";
-    final File configDir =
-        new File(TEST_HOME() + File.separator + "collection1" + File.separator + "conf");
+    final Path configDir = TEST_HOME().resolve("collection1").resolve("conf");
```

Comment:
LOVE THIS.  It exemplifies the improvements in this PR, and the sad state of half/partial usage of NIO before that so needed some love.